charts/trino/values.yaml

```diff
@@ -21,11 +21,18 @@ server:
     path: /etc/trino
     http:
       port: 8080
+    httpsServer:
+      enabled: false
+      port: 8443
+      keystore:
+        path: "/usr/local/certs/clustercoord.pem"
```

Comment:
I don't think any default value would be a good default here? should we leave this empty by default?